### DISCUSSION TITLE
Two classes with the same name java

### DISCUSSION THREAD

**Discussion by WojciechKorban** [javers/javers#1312] [Q&A]
Hi everyone, I have encountered a situation where I have two classes with the same name, 'Employee', but they exist in different locations. One is in a common library, and the other is in my module. Both classes are annotated with '@TypeName("Employee")'.

Unfortunately, I cannot make changes to the class in the common library. However, I want to designate the 'Employee' class in my codebase as the primary one and give it preference when deserializing objects.

Is there a way to achieve this and ensure that the 'Employee' class from my codebase is used during deserialization?

**Comment by bartoszwalacik** [comment]
You can override annotations config with `JaversBuilder.registerEntity(...)`
